pkg/sql/logictest/testdata/logic_test/distsql_enum

```diff
@@ -1,4 +1,5 @@
 # LogicTest: 5node-default-configs !5node-metadata
+# cluster-opt: disable-span-configs
```

Comment:
Do we want to file an issue so that this can be removed when fixed?

Reply:
👍  https://github.com/cockroachdb/cockroach/issues/75282